.concourse/pipeline.yaml.gomplate

```diff
@@ -1119,6 +1119,10 @@ jobs:
     - put: semver.gke-cluster
       params: {bump: patch}
   - task: upgrade
+    params:
+      GKE_KEY: '((gke-suse-cap-json))'
+      GKE_PROJECT: '{{ if has $ "gke_project" }}{{ $.gke_project }}{{ else }}suse-225215{{ end }}'
```

Comment:
```suggestion
      GKE_PROJECT: {{ default "suse-225215" $.gke_project | quote}}
```

Reply:
:warning: gomplate __does not have__ a `default` function. Using this form fails when the value is not set.

Reply:
Hmm, I thought it did (according to [the docs](https://docs.gomplate.ca/functions/conv/#conv-default)). But you're right, I totally forgot this is gomplate and not sprig… :D Sorry about that!